tests/fixtures/ALPHABETICAL_PATHS_REQUIRE_NO_PATH_EMPTY_ARRAY.js

```diff
@@ -0,0 +1,4 @@
+require([
```

Comment:
we already have some fixtures for these EMPTY cases...double check whats already there so we avoid duplicating anything